tests/blackbox/scopes/test_bb_scopes_updates.py

```diff
@@ -453,3 +452,96 @@ def test_scopes_linkage_update(setup_scopez_server_action):
     assert l_r.status_code == 403
     assert l_r.text == 'This is rules custom response\n'
 
+# ------------------------------------------------------------------------------
+# test /update_bots endpoint
+# ------------------------------------------------------------------------------
+def test_update_bots_endpoint(setup_scopez_server_action):
+    # ------------------------------------------------------
+    # test for recieving a bot challenge with attack vector
+    # ------------------------------------------------------
+    l_url = G_TEST_HOST + '/update_bots'
+
+    l_file_path = os.path.dirname(os.path.abspath(__file__))
+    l_test_file = os.path.realpath(os.path.join(l_file_path,
+                                                '../../data/waf/conf/bots/0052-wHyMHxV7.bots.json'))
+    l_test_payload = ''
```

Comment:
one nit pick on style - remove all new lines and stick to code seperators instead of single line comment, it looks mixed in this test.. am not against single line comments, just to make the style look consistent across waflz.

Reply:
if code seperators looks too much could get rid of some comments that doesn't explain logic